### PR TITLE
767 feature request sqlmap

### PR DESCRIPTION
**Pull request by russo-christian**
## Tool Description:
SQLmap is a tool used to detect and exploit SQL injection vulnerabilities in web databases. 

### Tool Features:
The SQLmap tool will run a series of injections against the targeted database. It contains the following configurations:
1. Target URL: the URL or IP of the database to scan
2. Detection Level (1-5): determines how aggressive the scan will be (higher levels will take longer)
3. Risk Level (1-3): determines the types of tests performed, ranging from cautious to aggressive
4. Retrieve Database Banner: if true, will attempt to retrieve the banner of the target database
5. List All Databases: if true, will attempt to to enumerate all databases on the target
6. Retrieve Password Hashes: if true, will attempt to retrieve password hashes from the target database

### CODE REVIEW

**Review by basoreeper3007 — CHANGES_REQUESTED** [review]
Everything else looks good, well done. Just fix those inline comments and I'll approve for merging.

File: src/components/SQLmap/SQLmap.tsx [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+ * Represents the form values for the SQLmap component.
+ */
+interface FormValuesType {
+    targetURL: string;
+    detectionLevel: string; // Added detection level
+    riskLevel: string; // Added risk level
+    banner: boolean; // Added banner retrieval option
+    dbs: boolean; // Added database enumeration option
+    passwords: boolean; // Added password dump option
+}
```
[the remark this answers is not in the document]
Comment (russo-christian) [reply src/components/SQLmap/SQLmap.tsx]: changes made

File: src/components/SQLmap/SQLmap.tsx [hunk 2]
```diff
@@ -0,0 +1,233 @@
+import { useState, useCallback, useEffect } from "react";
+import { Button, Stack, TextInput, Checkbox, Select } from "@mantine/core"; // Added Select
```
[the remark this answers is not in the document]
Comment (russo-christian) [reply src/components/SQLmap/SQLmap.tsx]: changes made

**Review by basoreeper3007 — APPROVED** [review]
Much better! Good work!

**Pull request merged**